icepyx/core/orders.py

```diff
@@ -0,0 +1,201 @@
+from pathlib import Path
+from typing import Any, Dict, Union
+
+import earthaccess
+
+
+class DataOrder:
+    """
+    A class representing an order for Harmony data processing.
+
+    Attributes
+    ----------
+    job_id : str
+        The ID of the Harmony job.
+    type : str
+        The type of order (e.g., "subset").
+    granules : list
+        A list of granules included in the order.
+    harmony_client : object
+        The Harmony API client used for interacting with the service.
+
+    Methods
+    -------
+    id()
+        Returns the job ID of the order.
+    resume()
+        Resumes the order if it has been paused due to exceeding granule limits.
+    pause()
+        Pauses the order.
+    status()
+        Retrieves the status of the order.
+    download_granules(path, overwrite=False)
+        Downloads the granules associated with the order.
+    download(path, overwrite=False)
+        Downloads the granules, waiting until they are ready if necessary.
```

Comment:
Again, can probably remove these docs, to avoid double documentation on readthedocs at https://icepyx--657.org.readthedocs.build/en/657/user_guide/documentation/components.html#icepyx.core.orders.DataOrder
```suggestion
```